libpod/container_internal.go

```diff
@@ -976,6 +976,19 @@ func (c *Container) completeNetworkSetup() error {
 			}
 		}
 	}
+	// check if we have a bindmount for /etc/hosts
+	hostsBindMount := state.BindMounts["/etc/hosts"]
```

Comment:
Could do `_, hastBindMount := state.BindMounts...` to get a bool directly instead